browser/about_flags.cc

```diff
@@ -259,20 +259,21 @@
           kOsDesktop | kOsAndroid,                                             \
           FEATURE_VALUE_TYPE(brave_news::features::kBraveNewsFeature),         \
       },                                                                       \
-      {                                                                        \
-          "brave-news-v2",                                                     \
-          "Enable Brave News V2",                                              \
-          "Use the new Brave News UI and sources lists",                       \
-          kOsDesktop | flags_ui::kOsAndroid,                                   \
-          FEATURE_VALUE_TYPE(brave_news::features::kBraveNewsV2Feature),       \
-      },                                                                       \
       {                                                                        \
           "brave-news-peek",                                                   \
           "Brave News prompts on New Tab Page",                                \
           "Prompt Brave News via the top featured article peeking up from "    \
           "the bottom of the New Tab Page, after a short delay.",              \
           kOsDesktop,                                                          \
           FEATURE_VALUE_TYPE(brave_news::features::kBraveNewsCardPeekFeature), \
+      },                                                                       \
+      {                                                                        \
+          "brave-news-feed-update",                                            \
+          "Brave News Feed Update",                                            \
```

Comment:
Since this is a specific one-time update, shall we give it a version number, or time-base? Something like `"brave-news-feed-update-2023"`? In the same veing that we're removing BraveNewsV2 so that we don't bring along other users by accident.

Reply:
Whoops that should've stayed on my other branch, but good call!